src/components/Share/style.scss

```diff
@@ -6,28 +6,56 @@
   background-color: rgba(#000, 0.2);
 
   > .content {
-    @include grid(8);
-    @include offsetLeft(2);
-    @include offsetRight(2);
+    @include wrap-mobile;
+    @include clearfix;
+
+    @include large-up {
+      @include grid(8);
+      @include offsetLeft(2);
+      @include offsetRight(2);
```

Comment:
If you're using `grid(8)` and `offsetLeft(2)`, is there a need for `offsetRight`?